QuestionBot/QuestionBot/Model/QuestionMessageListener.cs

```diff
@@ -0,0 +1,71 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace QuestionBot.Model{
+
+    public class QuestionMessageListener : IMessageListener{
+
+        private IStore _questionDataStore;
+
+        public QuestionMessageListener(IStore store){
+            _questionDataStore = store;
+        }
+
+        public void ReceiveMessage(string input){
+            const string questionCommand = "/question";
+            const string answerTag = "/answer";
+
+            if (input.StartsWith(questionCommand)){
+                string questionText = input.Remove(0, questionCommand.Length);
+                questionText = questionText.TrimStart(' ');
```

Comment:
Note that if we just use [`Trim`](https://msdn.microsoft.com/en-us/library/t97s7bs3(v=vs.110).aspx) here then we don't have to check below if `questionText` is whitespace (instead we just check for empty string).